drivers/serial/Kconfig.nrfx

```diff
@@ -18,6 +19,11 @@ menuconfig UART_NRFX
 
 if UART_NRFX
 
+config NRF_UART_ASYNC_API
```

Comment:
Please rebase on master and instead of this option, use UART_0_ASYNC

Reply:
Done. Thanks for reviewing!